.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "open-payments-specifications"]
```

Comment:
We should add a note in the README (probably both in the root README and the local playground) that we need to initialize the submodule when using this repository. 
Something like this: https://github.com/interledger/open-payments-node?tab=readme-ov-file#local-development-environment